projboard/tests/test_user.py

```diff
@@ -38,9 +44,9 @@ def test_get_user_by_nickname(self, generate_user):
         assert generate_user.password == user.password
 
     @pytest.mark.django_db
-    def test_get_user_by_nickname_fail(self):
+    def test_get_user_by_nickname_fail(self, fake_user):
```

Comment:
This is not clear at all
Why do you use fake user? 

test_get_user_by_nickname_fail? 
What do you plan to test here?

Reply:
@Yarboa I'm testing get_user function using fake user which is not in the database at all